examples/hosts/app-integration/external-data/tests/customerService.test.ts

```diff
@@ -36,9 +38,17 @@ describe("mock-customer-service", () => {
 	 */
 	let customerService: Server | undefined;
 
+	/**
+	 * Datastore mapping of external resource id to its subscribers.
+	 *
+	 * @defaultValue A new new map will be initialized.
+	 */
+	let webhookCollection: Map<string, MockWebhook<ITaskData>>;
+
 	beforeEach(async () => {
 		externalDataService = await initializeExternalDataService({
 			port: externalDataServicePort,
+			webhookCollection,
 		});
```

Comment:
Do we need to reinitialize the `webhookCollection` to an empty map before passing it in? So subscriptions set up by one test don't show up in another one.

Reply:
None of the tests here need the webhookCollection, I'm simply passing it in since it is needed by the `initializeExternalDataService`.

Reply:
Oh hold on -- there is something missing here -- this test is still failing.